content/en/docs/ops/diagnostic-tools/istioctl-analyze/test.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# shellcheck disable=SC1090,SC2154
+# shellcheck disable=SC1090,SC2154,SC2001
```

Comment:
As a comment, I still see a lint failure when trying to run the lint locally so ended up trying to add the disable here and below. Lint is passing here, so not sure if I should remove one of the SC2001.